packages/sources/src/components/button/button.args.ts

```diff
@@ -82,6 +82,7 @@ export function buttonArgsMapper(a: ButtonArgs): Button | ButtonAnchor {
   switch(a.element) {
     case 'anchor':
       const args: ButtonAnchor = {
+        variant: a.legacy ? null : a.variant,
```

Comment:
Ik begrijp wat je doet, maar ik wil juist het tegenovergestelde: Geen legacy selectors meer tonen.

Of wil je dit nog doen zolang ze rood zijn? Wat mij betreft ook OK!

Reply:
Dat laatste is zeker het geval. Laten we dit opschonen zodra de deprecation uit de toolkit wordt gehaald.

Reply:
OK!